common.mk

```diff
@@ -14,11 +14,12 @@ OPTEE_OS_PATH			?= $(ROOT)/optee_os
 OPTEE_CLIENT_PATH		?= $(ROOT)/optee_client
 OPTEE_CLIENT_EXPORT		?= $(OPTEE_CLIENT_PATH)/out/export
 OPTEE_TEST_PATH			?= $(ROOT)/optee_test
-OPTEE_TEST_OUT_PATH 		?= $(ROOT)/optee_test/out
+OPTEE_TEST_OUT_PATH		?= $(ROOT)/optee_test/out
 HELLOWORLD_PATH			?= $(ROOT)/hello_world
 
 # default high verbosity. slow uarts shall specify lower if prefered
 CFG_TEE_CORE_LOG_LEVEL		?= 3
+CFG_TEE_BENCHMARK		?= y
```

Comment:
Sure, I just left it switched on just for this RFC PR
I'm going to set CFG_TEE_BENCHMARK to n by default in the final PR